pontoon/base/templates/locale_manage.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'locale.html' %}
+
+{% block title %}{{ super() }} &middot; Manage {% endblock %}
+
+{% block locale_subpage %}
+<form method="POST" action="{{ url('pontoon.locale.manage', locale.code) }}">
+  <input type="hidden" value="{{ csrf_token }}" name="csrfmiddlewaretoken">
+  <input type="hidden" name="translators">
+  <input type="hidden" name="managers">
+
+  <div class="clearfix">
+      <div class="user select available">
```

Comment:
This looks like a potential macro, since the only difference between these three is the classname, input id, label, and description. That might be a bad idea, though. It's hard to tell with HTML when a macro is a good idea.